src/ManageCourses.UcasCourseImporter/importer/Mapping/CourseLoader.cs

```diff
@@ -108,7 +108,7 @@ private Course LoadCourse(Provider provider, IEnumerable<UcasCourse> courseRecor
                 returnCourse.CourseSites = new Collection<CourseSite>(courseRecords.Select(x => new CourseSite
                 {
                     Site = allSites.Single(y => y.Provider?.ProviderCode == x.InstCode && y.Code == x.CampusCode),
-                    ApplicationsAcceptedFrom = x.CrseOpenDate,
+                    ApplicationsAcceptedFrom = DataHelper.GetDateTimeFromString(x.CrseOpenDate),
```

Comment:
Can we not deal with this closer to source? Ie from the importer?

Reply:
I considered this but rejected the idea as all the imported does is grab data from the spreadsheet. Better to put transform logic in here